package.json

```diff
@@ -64,8 +63,6 @@
     "nsp": "npm audit --production",
     "prettier": "prettier --write .",
     "prettier-check": "prettier --check .",
-    "cluster": "jest ./cluster_test.spec.ts --coverage",
```

Comment:
Should we have a command to run cluster test locally without npm packaging?

Reply:
I think it would be nice, but it requires either duplicating the cluster test file or scripting changing the import(s).